### DISCUSSION TITLE
How to change default theme to light in Form Builder

### DISCUSSION THREAD

**Discussion by azrlucky** [optimajet/formengine#35] [Q&A] [answered]
Hi Team, 

I am embedding the form builder in my application and headers have to be hidden. Only the actual form builder should be visible. Our application is light themed application in Electron but whenever a user installs the application in their machine and uses the form builder is viewed in the dark theme by default.

How can I configure the form builder to be visible only in light theme or by default in light theme?

**Comment by optimajet** [comment, marked as answer]
Hi @azrlucky,

We have a plan to add a property to control the theme.
For now, you can use this recipe - https://github.com/optimajet/formengine/discussions/26#discussioncomment-9933255.
